### ISSUE TITLE
Training Performance 

### ISSUE THREAD

**Issue by nomaad42** [ruiz-manuel/yolov7-pose-custom#3]
Thanks again for your active support of this repo! 

During training, after 9 epochs, my performance does not change. Just stuck at certain numbers (I attached the screenshot). 
Have you encountered this problem? Why do you think it happens? 
Which hyperparams or loss function should I change in order to tackle this problem? 
![image](https://user-images.githubusercontent.com/62198792/224478552-fa74df89-8c99-44f5-8408-5d625eadad16.png)

**Comment by nomaad42** [comment]
@ruiz-manuel 

Thank you very much for your code! 

I used this sigmas

`sigmas = torch.tensor([.71, .73, .88, .77, .76, .79, .79, .72, .72, .87], device=device) / 10.0`

and this loss function 

`lkpt += kpt_loss_factor*(((1 - torch.exp(-d/(s*(4*sigmas**2)+1e-9))) + 0.05*d)*kpt_mask).mean()`

It seemed to help
